spyder/plugins/ipythonconsole/confpage.py

```diff
@@ -241,14 +300,16 @@ def setup_page(self):
         run_lines_layout.addWidget(run_lines_edit)
         run_lines_group.setLayout(run_lines_layout)
 
-        # Run file Group
+        # Run file group
         run_file_group = QGroupBox(_("Run a file"))
-        run_file_label = QLabel(_("You can also run a whole file at startup "
-                                  "instead of just some lines (This is "
-                                  "similar to have a PYTHONSTARTUP file)."))
+        run_file_label = QLabel(_(
+            "Specify a Python file to execute at startup, "
+            "similar to <tt>PYTHONSTARTUP</tt>"
```

Comment:
```suggestion
            "Specify a Python file to execute at startup, similar to "
            "<tt>PYTHONSTARTUP</tt>"
```

Fix odd formatting (i.e. strings can take more space on the right).

Reply:
Its [SembBr](https://sembr.org/), easier for humans to parse, easier to maintain as the lines can be moved around easily and it avoids wasting loads of time reflowing whenever the string changes, and leads to smaller and more logical diffs. Has proven quite successful in our incremental adoption for CPython repos despite a some initial resistance from old timers.